README.md

```diff
@@ -1 +1,51 @@
-# Bookstore-App
\ No newline at end of file
+## Bookstore-App
+
+## Getting Started
+This is a project about a book store app.
```

Comment:
- [ ] Let's us understand clearly what you are building and what your app is capable of doing